pkg/components/openebs-operator/component.go

```diff
@@ -50,17 +96,22 @@ func (c *component) LoadConfig(configBody *hcl.Body, evalContext *hcl.EvalContex
 }
 
 func (c *component) RenderManifests() (map[string]string, error) {
-	tmpl, err := template.New("installer").Parse(operatorInstallerTmpl)
+	helmChart, err := util.LoadChartFromAssets("/components/openebs")
 	if err != nil {
-		return nil, errors.Wrap(err, "parse template failed")
+		return nil, fmt.Errorf("load chart from assets: %v", err)
 	}
-	var installerBuf bytes.Buffer
-	if err := tmpl.Execute(&installerBuf, c); err != nil {
-		return nil, errors.Wrap(err, "execute template failed")
+
+	values, err := util.RenderTemplate(chartValuesTmpl, c)
+	if err != nil {
+		return nil, fmt.Errorf("render chart values template: %v", err)
```

Comment:
For `fmt.Errorf`, `%w` should be used to wrap the error.
```suggestion
		return nil, fmt.Errorf("render chart values template: %w", err)
```